src/bloom.cpp

```diff
@@ -197,3 +208,38 @@ void CBloomFilter::UpdateEmptyFull()
     isFull = full;
     isEmpty = empty;
 }
+
+CRollingBloomFilter::CRollingBloomFilter(unsigned int nElements, double fpRate, unsigned int nTweak) :
+    b1(nElements*2, fpRate, nTweak), b2(nElements*2, fpRate, nTweak)
+{
+    // Implemented using two bloom filters of nElements each.
```

Comment:
Comment should be `two filters of nElements*2 each`, if I understand the above initialization correctly?